### ISSUE TITLE
Tests failing after building

### ISSUE THREAD

**Issue by PMassicotte** [jmbuhr/tree-sitter-pandoc-markdown#1]
I am running these commands:

```shell
git clone --recurse-submodules git@github.com:jmbuhr/tree-sitter-pandoc-markdown.git
cd tree-sitter-pandoc-markdown
npm install
npm run build
npm run test
```

But it seems that the tests are failing. If I read it correctly, this suggests that `tree-sitter-pandoc-markdown` parser is missing the implementation of external scanner functions.

```shell
❯ npm run test

> tree-sitter-pandoc-markdown@0.1.0 test
> node scripts/test.js

testing tree-sitter-pandoc-markdown
Missing required functions in the external scanner, parsing won't work without these!

  `tree_sitter_pandoc_markdown_external_scanner_create`
  `tree_sitter_pandoc_markdown_external_scanner_destroy`
  `tree_sitter_pandoc_markdown_external_scanner_serialize`
  `tree_sitter_pandoc_markdown_external_scanner_deserialize`
  `tree_sitter_pandoc_markdown_external_scanner_scan`

You can read more about this at https://tree-sitter.github.io/tree-sitter/creating-parsers#external-scanners

testing tree-sitter-pandoc-markdown-inline
Missing required functions in the external scanner, parsing won't work without these!

  `tree_sitter_pandoc_markdown_inline_external_scanner_create`
  `tree_sitter_pandoc_markdown_inline_external_scanner_destroy`
  `tree_sitter_pandoc_markdown_inline_external_scanner_serialize`
  `tree_sitter_pandoc_markdown_inline_external_scanner_deserialize`
  `tree_sitter_pandoc_markdown_inline_external_scanner_scan`

You can read more about this at https://tree-sitter.github.io/tree-sitter/creating-parsers#external-scanners
```

If I try to run the test a second time, I get:

```shell
❯ npm run test

> tree-sitter-pandoc-markdown@0.1.0 test
> node scripts/test.js

testing tree-sitter-pandoc-markdown
Error opening dynamic library "/home/filoche/.cache/tree-sitter/lib/pandoc_markdown.so"

Caused by:
    /home/filoche/.cache/tree-sitter/lib/pandoc_markdown.so: undefined symbol: tree_sitter_pandoc_markdown_external_scanner_create
testing tree-sitter-pandoc-markdown-inline
Error opening dynamic library "/home/filoche/.cache/tree-sitter/lib/pandoc_markdown_inline.so"

Caused by:
    /home/filoche/.cache/tree-sitter/lib/pandoc_markdown_inline.so: undefined symbol: tree_sitter_pandoc_markdown_inline_external_scanner_create
```

Any ideas on what is going on?

**Comment by PMassicotte** [comment]
If I look in `tree-sitter-markdown-inline/src/scanner.c` I can see it is implemented

```c
void *tree_sitter_markdown_inline_external_scanner_create() {
    Scanner *s = (Scanner *)malloc(sizeof(Scanner));
    deserialize(s, NULL, 0);
    return s;
}
```

I am guessing this is something with missing bindings somewhere.

**Comment by jmbuhr** [comment]
oh, looks like I lost some changes during making all the necessary renames. Yes, all `_markdown` in the function names of both `scanner.c` have to be replaced with `pandoc_markdown`.

**Comment by PMassicotte** [comment]
yes found that, pr coming

**Comment by PMassicotte** [comment]
Closed in 84db6e154efbd97a49f2516f38f4f379195aef98
